src/Diagnostics.DataProviders/TokenService/TokenServiceBase.cs

```diff
@@ -1,44 +1,45 @@
-﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
-using System.Threading.Tasks;
+﻿// <copyright file="TokenServiceBase.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// </copyright>
+
```

Comment:
is this something we have to do in every new class?

Reply:
The styles are enforcing this. It will just create a warning I believe if we don't include it